src/test/java/org/abstractj/kalium/crypto/UtilTest.java

```diff
@@ -31,4 +34,10 @@ public void testPrependZeros() throws Exception {
     public void testDataNull() {
         Util.checkLength(null, 3);
     }
+
+    @Test
+    public void testSodiumVersion() {
+        Assert.assertTrue(NaCl.sodium().sodium_version_string() + " did not match expected pattern.",
```

Comment:
I'd add static imports for such commonly used and familiar methods (like assertX, mock/verify, various matchers).